### ISSUE TITLE
Best practice tagging: errata

### ISSUE THREAD

**Issue by jrbowden** [daisy/ebraille#147]
In the best practice tagging document, I found the following minor errors:

1. Emphasis
Reserved names duplicate custom-2
Should read: ... custom-1 custom-2 custom-3 ...

Note: this is very much UEB. What do other braille coding standards have?

2. Emphasis
Example of a single word in italics:
The braille dots should start ⠨⠂ not ⠘⠂
i.e.:
A single word in italics
<p>⠠⠏⠗⠑⠎⠎ ⠞⠓⠑ <em>⠨⠂⠎⠟⠥⠁⠗⠑</em> ⠃⠥⠞⠞⠕⠝⠲</p>

3. Emphasis
Example using custom-1 emphasis:
If in UEB, the start of the braille should be ⠈⠼⠂ not ⠘⠂
i.e.
Use <em> with a class value for emphasis other than bold and italics.
<p>⠠⠏⠗⠑⠎⠎ ⠞⠓⠑ <em class="custom-1">⠈⠼⠂⠎⠟⠥⠁⠗⠑</em> ⠃⠥⠞⠞⠕⠝⠲</p>

4. Lists
Suggest we should also add the element ``li`` to the list of elements.
i.e.
Element(s):
ol ul dl li

(as li is used in the examples).

**Comment by egli** [comment]
> Suggest we should also add the element `li` to the list of elements.

I would not add the `li` element as it is just part of the actual list element (either `ol`, `ul` or `dl`.

In other words leave the Elements section of Lists as it is

**Comment by wfreeman-aph** [comment]
I'm with Christian on the question of the "li" tag but agree with the other changes and appreciate you finding them. If no one objects, I will make these changes, make a pull request, and then the issue can be closed once the change can be confirmed in the main document.

**Comment by jrbowden** [comment]
... in which case I think we need to change the section on tables as well because it mentions ``tr`` and ``td``.
I therefore maintain we should include ``li``.
Hope this helps.

**Comment by egli** [comment]
My understanding is that the document lists only the html elements that are used to markup the specific text and which can be enhanced with specific `class` attributes. So in the case of lists only `ol`, `ul` and `dl` have any class attributes.

With that logic, yes, I guess we should remove all elements from the tables section except for `table`

**Comment by wfreeman-aph** [comment]
@jrbowden makes a good point about `table` and `li` and between removing `tr`  and `td` or adding `li`, I am thinking it is better to be too verbose than not clear enough. I am now thinking we should add `li`. I get where you are coming from @egli and definitely think there is a logic to it but I do fear readers being confused or uncertain what we intend. If we are consistent and name all the elements involved, it seems less likely to confuse than leaving aspects implied or shown only in examples.

**Comment by skntkacm** [comment]
I have a question regarding the Emphasis you mentioned. Will it be possible to change the braille prefix for emphasis? For example we do not use ⠨⠂ in Slovakia. And emphasis in our rules always have an opening and closing combinations. If this will be possible to set up in some template within styles, then it does not matter now - I can set it up according to our rules later. But I would like to know if it will be possible also in Emphasis prefixes and style (opening and closing codes of Emphasis). 

I am sorry, if this is question I should have know the answer for, I got lost a bit in all CSS and HTML tags and even if I tried hard to follow, I am not enough skilled in this. 😊
Michal

**Comment by egli** [comment]
I would assume that the braille prefix, i.e. the emphasis indication will be part of the unicode braille contained in the html. In your case this will be unicode braille according to the Slovakian braille standard. The html `em` element is just for additional information.

So there is no need to "change" the braille prefix as the entire braille will be under your control. Ebraille is just about how this braille will be distributed.

**Comment by skntkacm** [comment]
Thank you for clarification. So then second part of my concern might be part of EBRF - how the emphasis will look like. In slovakia we have patterns for opening and closing of emphasis. So it is different than UEB, as far as I know. In our case, these patterns work like parentheses. For example:
if you have two words in the sentence underlined, the structure is:
opening of emphasis,first word, second word,closing of emphasis.
e.g. It is necessary you send us your response **until the end of the week.** 
⠠⠊⠞⠀⠊⠎⠀⠝⠑⠉⠑⠎⠎⠁⠗⠽⠀⠽⠕⠥⠀⠎⠑⠝⠙⠀⠥⠎⠀⠽⠕⠥⠗⠀⠗⠑⠎⠏⠕⠝⠎⠑⠀⠣⠆⠥⠝⠞⠊⠇⠀⠞⠓⠑⠀⠑⠝⠙⠀⠕⠋⠀⠞⠓⠑⠀⠺⠑⠑⠅⠲⠣⠆⠜⠀
So my question is also: Is it possible to change the structure of emphasis tagging?

Thank you.

**Comment by jrbowden** [comment]
Hi @skntkacm, your example would be coded as:

```It is necessary you send us your response until the end of the week.

<p>⠠⠊⠞⠀⠊⠎⠀⠝⠑⠉⠑⠎⠎⠁⠗⠽⠀⠽⠕⠥⠀⠎⠑⠝⠙⠀⠥⠎⠀⠽⠕⠥⠗⠀⠗⠑⠎⠏⠕⠝⠎⠑⠀<em>⠣⠆⠥⠝⠞⠊⠇⠀⠞⠓⠑⠀⠑⠝⠙⠀⠕⠋⠀⠞⠓⠑⠀⠺⠑⠑⠅⠲⠣⠆⠜</em></p>⠀
```
Hope this helps.
